mis_builder/models/aep.py

```diff
@@ -281,14 +284,20 @@ def do_queries(self, date_from, date_to,
                                         ['debit', 'credit', 'account_id'],
                                         ['account_id'])
             for acc in accs:
+                company = account_model.browse(acc['account_id'][0]).company_id
```

Comment:
But if target currency is CHF which is 0.5 rounding and we're getting amounts in EUR which rounding is 0.1, won't we miss some amounts? That could be problematic especially if there are many amounts with 0.1 or 0.2 ?